C#/Symbol List Structs.tmPreferences

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+	<key>name</key>
+	<string>Symbol List: Struct</string>
+	<key>scope</key>
+	<string>source.cs entity.name.struct</string>
+	<key>settings</key>
+	<dict>
+		<key>showInSymbolList</key>
+		<integer>1</integer>
+		<key>showInIndexedSymbolList</key>
+		<string>1</string>
+		<key>symbolTransformation</key>
```

Comment:
We generally don't prefix the symbol type in the Goto Symbol list, so I'd like to see this and the next three lines removed.

Reply:
Consider it done